packages/workflows/src/helper/workflow-export.ts

```diff
@@ -56,6 +56,7 @@ export const exportWorkflow = <TData = unknown, TResult = unknown>(
         resultFrom: defaultResult,
       }
     ) => {
+      resultFrom ??= defaultResult
```

Comment:
todo: will need @carlos-r-l-rodrigues to pitch in here

Reply:
Ahah ive just kept the default value, in case the workflow is exported with one, but you pass no default when running then the default was lost

Reply:
True, i ll add it